docs/docs/en/guide/parameter/built-in.md

```diff
@@ -2,47 +2,33 @@
 
 ## Basic Built-in Parameter
 
-<table>
-    <tr><th>Variable</th><th>Declaration Method</th><th>Meaning</th></tr>
-    <tr>
-        <td>system.biz.date</td>
-        <td>${system.biz.date}</td>
-        <td>The day before the schedule time of the daily scheduling instance, the format is yyyyMMdd</td>
-    </tr>
-    <tr>
-        <td>system.biz.curdate</td>
-        <td>${system.biz.curdate}</td>
-        <td>The schedule time of the daily scheduling instance, the format is yyyyMMdd</td>
-    </tr>
-    <tr>
-        <td>system.datetime</td>
-        <td>${system.datetime}</td>
-        <td>The schedule time of the daily scheduling instance, the format is yyyyMMddHHmmss</td>
-    </tr>
-</table>
+| Variable | Declaration Method | Meaning |
+| ---- | ---- | -----------------------------| 
+| system.biz.date | ${system.biz.date} | The day before the schedule time of the daily scheduling instance, the format is yyyyMMdd |
+| system.biz.curdate | ${system.biz.curdate} | The schedule time of the daily scheduling instance, the format is yyyyMMdd |
+| system.datetime | ${system.datetime} | The schedule time of the daily scheduling instance, the format is yyyyMMddHHmmss |
```

Comment:
the parameter doc already change in #10659 , you should remvoe those change in this PR